src/modules/cmdpal/WindowsCommandPalette/TagViewModel.cs

```diff
@@ -38,11 +52,11 @@ public Windows.UI.Color Color
 
     // TODO! VV These guys should have proper theme-aware lookups for default values
     // All this code is exceptionally terrible, but it's just here to keep the POC app running at this point.
-    internal Brush BorderBrush => new SolidColorBrush(Color);
+    internal Brush BorderBrush => new SolidColorBrush(Foreground);
 
-    internal Brush TextBrush => new SolidColorBrush(Color.A == 0 ? Windows.UI.Color.FromArgb(255, 255, 255, 255) : Color);
+    internal Brush TextBrush => new SolidColorBrush(Foreground.A == 0 ? Windows.UI.Color.FromArgb(255, 255, 255, 255) : Foreground);
```

Comment:
i have not tested the POC app with this change, but c'est la vie

Reply:
We could just leave this as a reference to the Background for the original behavior as we think we're close to switching anyway?